www/README.md

```diff
@@ -7,7 +7,7 @@ Run locally with:
 - `yarn install`
 - `npm run develop`
 
-See the full contributing instructions at https://www.gatsbyjs.org/contributing/how-to-contribute/.
+See the full contributing instructions at [Gatsby contribute](https://www.gatsbyjs.org/contributing/how-to-contribute/).
```

Comment:
I'd recommend leaving this as-is, or removing the punctuation.
```suggestion
See the full contributing instructions at https://www.gatsbyjs.org/contributing/how-to-contribute/
```

Reply:
changes updated